stone_age/player_board/player_tools.py

```diff
@@ -9,7 +11,7 @@ def __init__(self) -> None:
         self._used_tools: List[bool] = [False] * 3
 
     def new_turn(self) -> None:
-        self._used_tools = [False for _ in range(len(self._tools))]
+        self._used_tools = [False for _ in range(3)]
 
     def add_tool(self) -> None:
         minimal_tool: int = min(self._tools)
```

Comment:
If `_tools=[3, 3, 3, 2]`, single-use tool would be falsely chosen leading to no effect instead of adding a tool